bots/zephyr_mirror_backend.py

```diff
@@ -368,8 +382,7 @@ def process_notice(notice, log):
     if notice.format.startswith("Zephyr error: See") or notice.format.endswith("@(@color(blue))"):
         logger.debug("Skipping message we got from Zulip!")
         return
-    if (zephyr_class == "mail" and notice.instance.lower() == "inbox" and is_personal and
-            not options.forward_mail_zephyrs):
+    if (zephyr_class == "mail" and notice.instance.lower() == "inbox" and is_personal and not options.forward_mail_zephyrs):
```

Comment:
The Travis error (the E129 rule from PEP8) occurred on an [older commit](6f0423a3bb1692a6ea678fb831bfab1fd1946e83) where the second line wasn't indented so far, and if @anirudhjain75 now puts these the way they are after a6f91064a2b6aab50153afd80a5f4833c3a8ea85 the PEP8 tests should pass.